### PR TITLE
Add usaf tsv file

### PR DESCRIPTION
**Pull request by joseph-robertson**
If you want to change out sets of weather files from the national TMY set (e.g., EFS study used the AMY set), you only need to update `Location EPW.tsv` and `Weather File Type.tsv` instead of all tsv files that are dependent on `Location EPW.tsv`.

https://trello.com/c/W7UtbJSS/72-add-usaf-parameter-with-usaf-to-tmy3-and-usaf-to-amy-mappings

### CODE REVIEW

File: project_resstock_testing/housing_characteristics/Location EPW.tsv [hunk 1]
```diff
@@ -1,2 +1,217 @@
-Option=USA_AZ_Phoenix-Sky.Harbor.Intl.AP.722780_TMY3.epw	Option=USA_CA_Los.Angeles.Intl.AP.722950_TMY3.epw	Option=USA_GA_Atlanta-Hartsfield-Jackson.Intl.AP.722190_TMY3.epw	Option=USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw	Option=USA_TX_Houston-Bush.Intercontinental.AP.722430_TMY3.epw	Option=USA_WA_Seattle-Tacoma.Intl.AP.727930_TMY3.epw
-0.166666667	0.166666667	0.166666667	0.166666667	0.166666667	0.166666667
```
Comment (ejhw) [inline project_resstock_testing/housing_characteristics/Location EPW.tsv]: I wonder if we should consider a different format for these super sparse matrices. I think this works for now, but other options might be to have the "TMY3" option point to the S3 url and just avoid needing to assign a EPW file??

Comment (joseph-robertson) [reply project_resstock_testing/housing_characteristics/Location EPW.tsv]: Not sure I follow.

Comment (ejhw) [reply project_resstock_testing/housing_characteristics/Location EPW.tsv]: Sorry I meant to comment on `project_resstock_national/housing_characteristics/Location EPW.tsv` I made a note to discuss later: https://trello.com/c/fekESvE3

**Comment by ejhw** [comment]
@joseph-robertson When you make the tsv name changes, please add new tsv names to https://github.nrel.gov/nmerket/resstockweb/blob/master/resstockweb/data/housing_characteristics_categories.yaml (and preserve the old names for backwards compatibility).

**Comment by ejhw** [comment]
👍

**Comment by joseph-robertson** [comment]
@ejhw Is this ready to go?

File: docs/source/advanced_tutorial/updating_projects.rst [hunk 2]
```diff
@@ -30,6 +30,6 @@ Click "Update Project" after each icon click. Click "OK" when done.
...
+If you made changes to either the `Building Characteristics Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/BuildingCharacteristicsReport>`_ or `Simulation Output Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/SimulationOutputReport>`_ measures, you will need to make updates on the Outputs tab. To do this remove the measure(s) from the project and re-add them, making sure they occupy the original position in the workflow order. If you changed any ``housing_characteristics`` tsv filenames, you will additionally need to make a change (e.g., add a whitepace) to ``measure.rb`` of `Building Characteristics Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/BuildingCharacteristicsReport>`_. Then navigate to the Outputs tab, select the measure(s) you've re-added, and then choose the outputs to include. Click "OK" when done. These outputs will be included in the summary results csv file.
```
Comment (ejhw) [inline docs/source/advanced_tutorial/updating_projects.rst]: @joseph-robertson We should copy or link to the second half of this from the section on modifying probability distributions. https://resstock.readthedocs.io/en/latest/advanced_tutorial/modifying_probability_distributions.html

Comment (joseph-robertson) [reply docs/source/advanced_tutorial/updating_projects.rst]: @ejhw Take a look at my addition.

**Pull request merged**